client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/AbstractArmeriaCentralDogmaBuilder.java

```diff
@@ -82,6 +87,15 @@ public final B clientConfigurator(ArmeriaClientConfigurator clientConfigurator)
         return self();
     }
 
+    /**
+     * Sets the DNS query timeout for looking up Central Dogma server.
+     * If not set, it will use <a href="https://line.github.io/armeria/">Armeria</a> client built-in defaults.
+     */
+    public final B dnsQueryTimeout(@Nullable Duration dnsQueryTimeout) {
```

Comment:
Could you add `dnsQueryTimeoutMillis` as well because we always have the pair of setters for the time?
Also, could you remove `@Nullable` and delegate to the `dnsQueryTimeoutMillis` method, please?

Reply:
I don't get the need to use all available setters

Reply:
Can you please elaborate on the reason why having 2 setters here too is better?

Reply:
It's sort of our convention to provide the same API experience. 😄

Reply:
okay, understood. Let me add it.